gst/nnstreamer/tensor_filter/tensor_filter_common.c

```diff
@@ -1215,7 +1215,7 @@ gst_tensor_filter_detect_framework (const gchar * const *model_files,
     else if (g_str_equal (ext[0], ".bin") || g_str_equal (ext[0], ".xml"))
       detected_fw = g_strdup ("openvino");
     else if (g_str_equal (ext[0], ".tvn"))
-      detected_fw = g_strdup ("srnpu");
+      detected_fw = g_strdup ("trixengine");
```

Comment:
Is this safe?
@dongju-chae

Reply:
@dongju-chae 
Which do you prefer, **trix_engine** or **trixengine**?

Reply:
To rename this, you need to update subplugin.
I'd suggest to add "aliasing" for tensor-filter framework names. (with .ini configurations)

Reply:
I think `trix-engine` looks better (similar to `npu-engine`). I will update the subplugin when this PR is available in Tizen emulator.